tests/test_views_rest.py

```diff
@@ -35,7 +35,148 @@
 from flask_security import url_for_security
 from six import BytesIO
 
-from invenio_deposit.api import Deposit
+
+def test_edit_deposit_1(app, db, es, users, location, deposit, write_token):
```

Comment:
> "write as many words you can" 

That's not what I called for :smile: and it's not what our coding standards say either.  Obviously one should find reasonable bounds on the length.

Here are examples of renaming I had in mind:

- `test_delete_deposit_1` -> `test_delete_deposit_owner`
- `test_delete_deposit_2` -> `test_delete_deposit_anonymous`
- `test_delete_deposit_3` -> `test_delete_deposit_unauthorised`
- `test_delete_deposit_4` -> `test_delete_deposit_oauth_good`
- `test_delete_deposit_5` -> `test_delete_deposit_oauth_bad`

The latter names are much more readable, when a test fails you know immediately what went wrong without checking what 5 may mean.  And the names still retain reasonable length, I think.